addons/jam/CfgWeapons.hpp

```diff
@@ -22,14 +22,14 @@ class CfgWeapons {
 
     class GrenadeLauncher;
     class UGL_F: GrenadeLauncher {
-        magazineWell[] += {"CBA_40mm_M203","CBA_40mm_EGLM"}; // empty in vanilla
+        magazineWell[] += {"CBA_40mm_M203","CBA_40mm_EGLM"};
     };
 
     class arifle_MX_Base_F: Rifle_Base_F {
         magazineWell[] += {"CBA_65x39_MX","CBA_65x39_MX_XL"};
 
         class GL_3GL_F: UGL_F {
-            magazineWell[] += {"CBA_40mm_3GL","CBA_40mm_M203","CBA_40mm_EGLM"}; // [] in vanilla
+            magazineWell[] = {"UGL_40x36","3UGL_40x36","CBA_40mm_3GL","CBA_40mm_M203","CBA_40mm_EGLM"}; // just list all wells because of += limitations
```

Comment:
```suggestion
            magazineWell[] += {"CBA_40mm_3GL"};
```
Am I missing something? If I use that ^, result ingame is:
`magazineWell[] = {"CBA_40mm_M203","CBA_40mm_EGLM","CBA_40mm_3GL"};`
Which is exactly what we want, no?

Reply:
Missing `"UGL_40x36","3UGL_40x36"`, I think we use `+=` so we don't break compat with mods using vanilla magwells

Reply:
Ah, now I understand the problem.